grails-app/views/stockMovement/list.gsp

```diff
@@ -36,13 +36,14 @@
         <div class="right">
 
             <div class="button-group">
-                <g:link controller="stockMovement" action="list" params="['createdBy.id':session?.user?.id]" class="button">
+                <g:set var="isPrimary" value="${params.createdBy}"/>
+                <g:link controller="stockMovement" action="list" params="['createdBy.id':session?.user?.id]" class="button ${isPrimary?'primary':''}">
```

Comment:
you did not used a separate variable for the All button, I think you can use params.createdBy directly here too